topics/epigenetics/tutorials/ewas-suite/slides.html

```diff
@@ -15,7 +15,7 @@
 ## Overview
 
 - Applications of EWAS
-- The Data (450k what is it?)
```

Comment:
@kpbioteam on top can you add your name as well and `questions:` etc as metadata?

Reply:
I have updated name and add metadata.yaml